amazon-s3/5.6/modules/ROOT/pages/amazon-s3-connector-config-topics.adoc

```diff
@@ -9,6 +9,151 @@ In the `Create Object` operation, set the `Content Length` to a value greater th
+
+[source,json,linenums]
+----
+{
+  "Version": "2012-10-17",
+  "Id": "arn:aws:sqs:REGION:ACCOUNT_ID:QUEUE_NAME/SQSDefaultPolicy",
+  "Statement": [
+    {
+      "Sid": "Sid1593770403887",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "SQS:SendMessage",
+      "Resource": "arn:aws:sqs:REGION:ACCOUNT_ID:QUEUE_NAME",
+      "Condition": {
+        "ArnLike": {
+          "aws:SourceArn": "arn:aws:s3:::BUCKET_NAME"
+        }
+      }
+    }
+  ]
+}
+----
+
+[[queue-created-by-source]]
+== Queue Created by Source Operation
+
+If the queue is created by the source operation, the queue is created with the prefix `MULE-S3-TRIGGER`. 
+
+In this scenario, you must have permission to create a queue and to also set the queue attributes:
+
+[source,json,linenums]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "sqs:DeleteMessage",
+                "sqs:ReceiveMessage",
+                "sqs:GetQueueAttributes",
+                "sqs:CreateQueue",
+                "sqs:SetQueueAttributes"
+            ],
+            "Resource": "arn:aws:sqs:REGION:ACCOUNT_ID:MULE-S3-TRIGGER-*"
+        }
+    ]
+}
+----
+
+When the queue is created by the source operation, an S3 policy is added to the queue when the S3 object is created in the bucket, in the following format:
+
+
+[source,json,linenums]
+----
+{
+  "Version": "2012-10-17",
+  "Id": "example-ID",
+  "Statement": [
+    {
+      "Sid": "example-statement-ID",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "*"
+    },
+      "Action": "SQS:SendMessage",
+      "Resource":QUEUE_ARN,
+      "Condition": {
+        "ArnLike": {
+          "aws:SourceArn": "BUCKET_ARN"
+        }
+      }
+    }
+  ]
+}
+----
+
+== S3 Policy Bucket
+
+For the S3 policy, we use this form for both cases:
+
+[source,json,linenums]
+----
+{
+    "Version": "2012-10-17",
+    "Id": "Policy1593761427184",
+    "Statement": [
+        {
+            "Sid": "Stmt1593760119344",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::ACCOUNT_ID:user/test"
+            },
+            "Action": "SPECIFIC_ACTION",
+            "Resource": "arn:aws:s3:::BUCKET_NAME/*"
+        },
+        {
+            "Sid": "Stmt1593760259223",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::ACCOUNT_ID:user/test"
+            },
+            "Action": ["s3:PutBucketNotification", "s3:GetBucketNotification"],
+            "Resource": "arn:aws:s3:::BUCKET_NAME"
+        }
+    ]
+}
+----
+
+In this example, the *SPECIFIC_ACTION* is `s3:PutObject` or `s3:DeleteObject`, which is based on the specified source operation.
+
 == Next Step
 
 After you understand how to configure a master key and provide credentials, you can try out the xref:amazon-s3-connector-examples.adoc[Example].
```

Comment:
It wasn't until I got to this sentence that I realized that the two tasks here were
1. Configure master key
2. Provide credentials
Should those be explicitly stated using subheadings above, and the topic arranged around this?